js/index.js

```diff
@@ -635,6 +659,20 @@ $(function() {
 
                           myconsole.warn("Unable to load JS bundle", err);
                         });
+                      } else if ($('input[name=backend_inputs]').filter(':checked').val() === "mathbox") {
+                           $.when($.get("js/mathbox-bundle.js"),
+                                  $.get(backend.endpoint + "/bundle")
+                                ).done(function(mathboxBundle, bundle) {
+
+                           var replaced = bundle[0].replace(/require\("Data.Either"\)/g, "PS['Data.Either']")
+                                                   .replace(/require\("Data.Maybe"\)/g, "PS['Data.Maybe']")
+                                                   .replace(/require\("Mathbox.Field"\)/g, "PS['Mathbox.Field']")
```

Comment:
Good to mention, I have used these `require`'s explicitly in `js`:
https://github.com/rintcius/purescript-mathbox/blob/master/src/Mathbox/Mathbox.js#L6-L8

Could this be due to a mismatch in what `pulp browserify` does & how this bundle was created?